cmake_modules/FindProtobufGoby.cmake

```diff
@@ -133,7 +133,9 @@ find_path(PROTOBUF_INCLUDE_DIR google/protobuf/service.h)
 list(APPEND ALL_PROTOBUF_INCLUDE_DIRS "-I${PROTOBUF_INCLUDE_DIR}")
```

Comment:
Yeah it seems to me this may be the only line that is needed. Perhaps just delete 135 and 136? That doesn't cause any problems for me